disk/conf.yaml.default

```diff
@@ -41,3 +41,8 @@ instances:
     # get metrics for all partitions. use_mount should be set to yes (to avoid
     # collecting empty device names) when using this option.
     # all_partitions: no
+    #
+    # The (optional) device_tag_re parameter will instruct the check to apply additional tags
```

Comment:
tags could be coma separated, this should be said in the comment and show in the example.